ansible-playbook/transmute.yml

```diff
@@ -2,7 +2,7 @@
 - name: transmute
   vars:
     fix_dotfiles: true
-    transmute_kube_version: "v1.9.4"
+    transmute_kube_version: "v1.10.0"
```

Comment:
Change to Kubernetes 1.10.0. Kubernetes 1.9.4 was causing a failure to schedule minikube addons "storage-provisioner" and "dashboard" on macOS OS X 10.13.